README.md

```diff
@@ -1 +1,124 @@
-# openlxp-xia
\ No newline at end of file
+
+# OPENLXP-XIA
+# What is it?
+OpenLXP XIA is a Python package that provides the validation functionality of extracted metadata from the source. In addition, it helps transform metadata into target metadata and further load it into indexing services. The OpenLXP XIA package does not include the metadata extraction process because extraction methods can be different for different sources. But once metadata gets extracted from the source, OpenLXP-XIA continues the V-T-V-L  (Validate-Transform-Validate-Load) cycle. 
+
+The schema files used for validation can be placed on the schema server. Currently, OpenLXP-XIA uses AWS S3 buckets as a schema server. 
+
+Below are the workflow which are performed by the OpenLXP-XIA after package installation.
+
+
+# Workflows
+The OpenLXP-XIA implements five core workflows after extracting metadata from the Specifiec source, as follows:
+
+1. `Validate`: Compares extracted learning experience metadata against the configured source metadata reference schema stored in the Experience Schema Service (XSS).
+
+2. `Transform`: Transforms extracted+validated source learning experience metadata to the configured target schema using the "XSR-to-Target" transformation map stored in the Experience Schema Service (XSS)
+
+3. `Validate`: Compares transformed learning experience metadata against the configured target metadata reference schema stored in the Experience Schema Service (XSS).
+
+4. `Load`: Pushes transformed and validated learning experience metadata to the target Experience Index Service (XIS) for further processing.
+
+5. `Log`: Records error, warning, informational, and debug events which can be reviewed and monitored.
+
+# Prerequisites
+`Python >=3.7` : Download and install python from here [Python](https://www.python.org/downloads/).
+
+
+## Environment Variables
+
+To run this package along with your project, you will need to add the following environment variables to your .env file
+
+
+`BUCKET_NAME` - S3 Bucket name where schema files are stored
+
+`AWS_ACCESS_KEY_ID` - AWS access keys
+
+`AWS_SECRET_ACCESS_KEY` - AWS access password
+
+`AWS_DEFAULT_REGION` - AWS region
+
+
+# Installation
+
+    $ python -m pip install OpenLXP-XIA (use the latest package version)
+
+Add OpenLXP-XIA in the setting.py in your project.
+
+INSTALLED_APPS = [
+        ...
+        
+        'openlxp_xia',
+        
+        ....
+]
+
+# Configuration
+
+1. On the Admin page, log in with the admin credentials 
+
+3. `Add xis configuration`: Configure Experience Index Services (XIS): 
+
+`Xis metadata api endpoint`: API endpoint for XIS where metadata will get stored.
+
+Example:  
+`Xis metadata api endpoint`: http://localhost:8080/api/metadata/
+
+`Xis supplemental api endpoint`: API endpoint for XIS where supplemental metadata will get stored.
```

Comment:
@SHGODE  Formatting for this section is incorrect.  It should be indented like the xia configuration section (lines 76 - 86) is.

Reply:
can you  check if it's still the same! Markup shows its same as 76-86. Preview it bit weird

Reply:
looks good now!